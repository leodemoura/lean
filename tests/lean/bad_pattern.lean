constant P : nat → Prop

lemma tst₁ [forward] : ∀ x, (: P x :) := -- Fine
sorry

lemma tst₂ [forward] : ∀ x, P (: x :) := -- Error
sorry
