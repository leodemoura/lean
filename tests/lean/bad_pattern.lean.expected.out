bad_pattern.lean:6:33: error: invalid pattern hint, pattern must be applications
